docs/OPERATIONS.md

```diff
@@ -0,0 +1,101 @@
+# Deploy a Strategy / Vault for V2
+
+NOTE: This repo is encouraged to create multiple scripts for governance and dev multisig execution of complex transactions
+https://github.com/iearn-finance/chief-multisig-officer
+
+**IMPORTANT**: you should **NOT** create a new release with a test vault
+
+## Before deploying
+- [ ] Coordinate with Core Dev strategist for getting a review in board https://github.com/orgs/iearn-finance/projects/5
+- [ ] Peer review completed by:
+    - yearn.rocks: at least 2 strategists
+    - Experimental: at least 1 strategist + 1 core dev in board 
+- [ ] Check if want token has a deploy vault already (>=v0.3.0) and coordinate to use that first if possible.
+- [ ] (only if a new vault is needed) Deploy using the registry.
```

Comment:
Question: Given the room to make errors in parameters and the need for us to verify that those are correct in any case, should the Strategist really be doing the vault deployment, or is it better this is handled by dedicated people?

Reply:
It's a great question. I've added this to my list of tasks to revisit when we have all the process documented. I'd like to have the current process, with a nice amount of details and then we can improve it. What do you think?